openedx/core/djangoapps/user_api/preferences/tests/test_views.py

```diff
@@ -104,14 +103,14 @@ def test_get_preferences(self, api_client, user):
     @ddt.unpack
     def test_patch_unknown_user(self, api_client, user):
         """
-        Test that trying to update preferences for a user who does not exist returns a 404.
+        Test that trying to update preferences for a user who does not exist returns a 403.
```

Comment:
Why does get return 404 for staff, but this returns 403?  Do we need the ddt data for cases where everyone gets the same?  Why doesn't the bookmark get test for staff/404 combo, or is this a unique situation?